test/unit/web_framework/test_cluster_base_handler.py

```diff
@@ -37,7 +37,7 @@ def test_set_default_headers_should_not_set_origin_header(self, request_origin,
     )
     def test_set_default_headers_should_set_origin_header(self, request_origin, cors_conf_value):
         mock_application = MagicMock(spec_set=ClusterApplication())
-        mock_request = MagicMock(spec=tornado.httpserver.HTTPRequest, headers={})
+        mock_request = MagicMock(headers={})
```

Comment:
These had to be changed because of an incompatible change in the implementation of `tornado.web.RequestHandler` that made it not work with this mock in the same way. We don't do much with this mock instance anyway, so removing the spec is not a big deal.